tests/__mocks__/styles-mock.ts

```
@@ -0,0 +1,2 @@
+// Mock CSS modules
+export default {} 
```

ESLint

[error] Trailing spaces not allowed.

(@stylistic/no-trailing-spaces)